target/capstone-1/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Another appengine-web.xml?

Reply:
This file is auto generated by the maven appengine.